tasks/copy_to_glacier/test/unit_tests/test_copy_to_glacier.py

```diff
@@ -28,12 +28,7 @@
     _OUTPUT_VALIDATE = fastjsonschema.compile(json.loads(raw_schema.read()))
 
 
-class TestCopyToGlacierHandler(TestCase):
-    """
-    Test copy to glacier handler
-    This will test if the function of test copy to glacier work as expected
-    """
-
+class TestCopyToGlacier(TestCase):
```

Comment:
`TestCopyToArchive(TestCase):`? Thinking we should still have a doc string to tell us what this test is about.

Reply:
Added doc string. Waiting for rename until https://bugs.earthdata.nasa.gov/browse/ORCA-537